### PR TITLE
(fix) add conditional to starting ladders, update insertion to be rep…

### PR DESCRIPTION
**Pull request by mark-antony1**
…lacement

### CODE REVIEW

File: merkato/utils/database_utils.py [hunk 1]
```diff
@@ -141,7 +141,21 @@ def exchange_exists(exchange):
...
+    except Exception as e:
+        print(str(e))
+    finally:
+        c = conn.cursor()
+        c.execute('''SELECT * FROM merkatos WHERE exchange_pair = ?''', (UUID,))
+        result = len(c.fetchall())
+
+        conn.commit()
+        conn.close()
+        return result > 0
```
Comment (nasaWelder) [inline merkato/utils/database_utils.py:158]: result, itself, is sufficiently boolean

File: merkato/merkato.py [hunk 2]
```diff
@@ -26,7 +27,8 @@ def __init__(self, configuration, coin, base, spread, coin_reserve, base_reserve
         self.history = self.exchange.get_my_trade_history() # TODO: Reconstruct from DB
         self.bid_reserved_balance = coin_reserve
         self.ask_reserved_balance = base_reserve
-        self.distribute_initial_orders(base_reserve, coin_reserve)
+        if merkato_does_exist == False:
```
Comment (nasaWelder) [inline merkato/merkato.py]: if not merkato_does_exist:

File: merkato/utils/database_utils.py (hunk 1 above)
Comment (mark-antony1) [reply merkato/utils/database_utils.py:158]: I prefer explicit checking

File: merkato/merkato.py (hunk 2 above)
Comment (mark-antony1) [reply merkato/merkato.py]: agreed

**Review by needmoney90 — APPROVED** [review]

**Pull request merged**